community/security/src/test/java/org/neo4j/server/security/auth/FileUserRepositoryTest.java

```diff
@@ -77,7 +83,7 @@ public void shouldStoreAndRetriveUsersByName() throws Exception
     {
         // Given
         FileUserRepository users = new FileUserRepository( authFile, NullLogProvider.getInstance() );
-        User user = new User( "jake", Credential.INACCESSIBLE, true );
+        User user = new User( "jake", "admin", Credential.INACCESSIBLE, true );
```

Comment:
As above, I feel like in community there is no concept of groups. Should we be testing for this here?